### PR TITLE
add new performance improvement

### PR DESCRIPTION
**Pull request by WeizhuoZhang-intel**
- [x] Add new performance improvement
- [x] Add new fixed models
- [x] Fix issue https://github.com/chuanqi129/inductor-tools/issues/40 

**Excel:**
image

**Archive:**
image

**Email:**
image

### CODE REVIEW

File: scripts/modelbench/report.py [hunk 1]
```diff
@@ -64,6 +64,7 @@
 
 new_performance_regression=pd.DataFrame()
 new_failures=pd.DataFrame()
+new_performance_improvement=pd.DataFrame()
```
Comment (chuanqi129) [inline scripts/modelbench/report.py:67]: Please add a new_fixes model list also for those crash models in reference but fixed in target.

**Comment by WeizhuoZhang-intel** [comment]
Fixed models added:
image

**Comment by WeizhuoZhang-intel** [comment]
Fixed first line of performance regression was ignored issue:
https://github.com/chuanqi129/inductor-tools/issues/40
image

**Review by chuanqi129 — APPROVED** [review]
LGTM. Please merge it after verification by test

**Pull request merged**